lisa/sut_orchestrator/azure/features.py

```diff
@@ -1217,8 +1217,17 @@ def get_raw_data_disks(self) -> List[str]:
     def get_all_disks(self) -> List[str]:
         # /sys/block/sda = > sda
```

Comment:
1. Should the examples follow Linux patterns?
2. Move all patterns out of the method like other places.

Reply:
I have fixed first comment. For second, I feel moving regex out will make the testcase more complicated since we will add more lines to deal with different regex variable name.